doc/read-the-docs-site/reference/cardano/language-changes.rst

```diff
@@ -53,6 +53,48 @@ Vasil
 
 All of the built-in types and functions from ``PlutusV1`` were added to ``PlutusV2``.
 
-The following built-in function was added to ``PlutusV2`` only (i.e., it is not available in ``PlutusV1``).
+The following built-in function was added to ``PlutusV2`` only (ie, it is not available in ``PlutusV1``).
 
 - ``serializeData`` (proposed in `CIP-42 <https://cips.cardano.org/cips/cip42/>`_)
+
+PlutusV3
+~~~~~~~~~
+
+Plutus and cryptography teams at IOG, in collaboration with `MLabs <https://mlabs.city/>`_, continue to develop Plutus capabilities. Starting with the release of `Cardano node v.8.8.0-pre <https://github.com/IntersectMBO/cardano-node/releases/tag/8.8.0-pre>`_, ``PlutusV3`` is available on `SanchoNet <https://sancho.network/>`_, introducing the Cardano community to governance features from `CIP-1694 <https://cips.cardano.org/cip/CIP-1694#goal>`_ in a controlled testnet environment. 
+
+``PlutusV3`` is the new ledger language that enhances Plutus Core's cryptographic capabilities, offering the following benefits for the smart contract developer community: 
+
+- Providing an updated script context that will let users see `CIP-1694 <https://cips.cardano.org/cip/CIP-1694#goal>`_  governance-related entities and voting features
+- Interoperability between blockchains
```

Comment:
I'd remove these:
```
- Interoperability between blockchains
- Advanced Plutus primitives
- Well-known and optimal cryptographic algorithms
- Support for porting of smart contracts from Ethereum
- Creating sidechain bridges
```

and simply say "new cryptographic and bitwise primitives`. These primitives are already explained below, and it is unclear which specific primitive you are talking about when you say "Creating sidechain bridges" etc.